### ISSUE TITLE
Subscriptions

### ISSUE THREAD

**Issue by Protektor-Desura** [alexta69/metube#113]
It would be nice to have support for a subscription. So as new videos came out for a content creator or whatever then it would check every 24 hours for a new video and automatically download it for you.

I'm thinking of things like Critical Role and other gaming videos and board game review channels  where they do a daily or weekly update of a new show.

**Comment by PikuZheng** [comment]
Yes this idea is great. I've been using IFTTT to notify ytdl downloads when a subscription is updated. 
But honestly I don't think subscription is something ytdl should do

**Comment by Protektor-Desura** [comment]
Well this is something that one of the other ytdl's does but they don't have a web browser plugin to make it easier to download videos from your browser automatically.

https://github.com/Tzahi12345/YoutubeDL-Material
https://github.com/Tzahi12345/YoutubeDL-Material/blob/master/docker-compose.yml

You can see there, that there is a specific directory in the docker compose for storing subscriptions. So if metube had subscriptions then it would be be a replacement for YoutubeDL-Material since there are no browser plugins for it.

**Comment by alexta69** [comment]
I guess you're thinking about something like the *rr programms (Radarr, Sonarr etc.). I see MeTube more as a downloader (think a BitTorrent client or Sabnzbd/nzbget), less as a subscription manager like the *rr apps. So I think it would make sense to have a separate app for that.

**Comment by Protektor-Desura** [comment]
I would like to be able to download all of these shows automatically

Critical Role          https://www.youtube.com/c/criticalrole
Watch It Played    https://www.youtube.com/channel/UCGK9n7svoIjuaQfRIBJXkqQ
Dana Loesch         https://www.youtube.com/channel/UCMgU2c86gIV4s408gNwl-hQ
Steven Crowder    https://www.youtube.com/channel/UCIveFvW-ARp_B_RckhweNJw
Glenn Beck           https://www.youtube.com/user/glennbeck
Joe Rogan            https://www.youtube.com/channel/UCzQUP1qoWDoEbmsQxvdjxgQ
Ubuntu On Air     https://www.youtube.com/c/UbuntuOnAir
Paul Hibbert         https://www.youtube.com/c/PaulHibbert
Linux Tech Tips     https://www.youtube.com/channel/UCXuqSBlHAE6Xw-yeJA0Tunw

All of these put out at least one new show a week and many are daily.

**Comment by ldexterldesign** [comment]
https://github.com/meeb/tubesync
